### PR TITLE
Installation: FindGMP also searches for debug lib

### LINKED ISSUE #5567: The program can't start because gmp.dll is missing from your computer. Try reinstalling the program to fix this problem.
## Issue Details
I downloaded cgal:x64-windows via vcpkg and successfully generated the project with cmake-gui.

However, when I run the code "simple_mcfskel_sm_example.cpp" in "Debug" modus (VS2019), I got the error message: "The program can't start because gmp.dll is missing from your computer. Try reinstalling the program to fix this problem." When I switch to "Realease" modus, everything is fine.

I checked the file under the directory "_<project_dir>/build/Debug_". Thers is no "_gmp.dll_" but only a "_gmpd.dll_". Actually the "_gmp.dll_" is under "_<project_dir>/build/Realease_".

What should I do to copy "_gmp.dll_" automaticlly in DEBUG modus?

## Source Code
I am using the example code : "_simple_mcfskel_sm_example.cpp_" .

## Environment

* Operating system (Windows 10, 64 bit):
* Compiler: cmake-gui
* Release or debug mode: DEBUG
* Specific flags used (if any): all packages are up to date
* CGAL version:5.2.1
* Boost version: 1.75.0
* Other libraries versions if used (Eigen, TBB, etc.):

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes
FindGMP.cmake now searches for gmpd as a debug library for gmp. This fixes the vcpkg integration bug.
## Release Management

* Affected package(s):Installation
* Issue(s) solved (if any): fix #5567

### CODE REVIEW

File: Installation/cmake/modules/FindGMP.cmake [hunk 1]
```diff
@@ -31,15 +31,24 @@ if( NOT GMP_in_cache )
...
-
-  if ( GMP_LIBRARIES )
+	
+	find_library(GMP_LIBRARY_DEBUG NAMES gmpd libgmp-10 mpir
+    HINTS ENV GMP_LIB_DIR
+          ENV GMP_DIR
+          ${CGAL_INSTALLATION_PACKAGE_DIR}/auxiliary/gmp/lib
+    PATH_SUFFIXES lib
+    DOC "Path to the Debug GMP library"
+    )
```
Comment (lrineau) [inline Installation/cmake/modules/FindGMP.cmake:48]:
```suggestion
	find_library(GMP_LIBRARY_DEBUG NAMES gmpd gmp libgmp-10 mpir
    HINTS ENV GMP_LIB_DIR
          ENV GMP_DIR
          ${CGAL_INSTALLATION_PACKAGE_DIR}/auxiliary/gmp/lib
    PATH_SUFFIXES lib
    DOC "Path to the Debug GMP library"
    )
```

Comment (lrineau) [reply Installation/cmake/modules/FindGMP.cmake:48]: That way it can work if there is no `gmpd`.

File: Installation/cmake/modules/FindGMP.cmake [hunk 2]
```diff
@@ -31,15 +31,24 @@ if( NOT GMP_in_cache )
...
+	
+	set(GMP_LIBRARIES $<IF:$<CONFIG:Debug>,${GMP_LIBRARY_DEBUG},${GMP_LIBRARY_RELEASE}>)
```
Comment (lrineau) [inline Installation/cmake/modules/FindGMP.cmake:50]:
```suggestion
	set(GMP_LIBRARIES debug "${GMP_LIBRARY_DEBUG}" optimized "${GMP_LIBRARY_RELEASE}")
```

File: Installation/cmake/modules/FindGMP.cmake [hunk 3]
```diff
@@ -31,15 +31,33 @@ if( NOT GMP_in_cache )
...
+  if(IS_MULTI_CONFIG)
+  set(GMP_LIBRARIES debug ${GMP_LIBRARY_DEBUG} optimized ${GMP_LIBRARY_RELEASE})
+else()
+  if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
+    set(GMP_LIBRARIES ${GMP_LIBRARY_DEBUG})
+  else()
+    set(GMP_LIBRARIES ${GMP_LIBRARY_RELEASE})
+  endif()
+endif()
+   if ( GMP_LIBRARIES )
```
Comment (lrineau) [inline Installation/cmake/modules/FindGMP.cmake:60]: The indentation pattern is not right.

And the line 61 (https://github.com/CGAL/cgal/pull/5578/files#diff-8ea666e9d7c8d634a294f5f2747c2515644ead1c9154cbed00654b8b8030b1e7R61) cannot run correctly. You should remove that variable `GMP_LIBRARIES_DIR` from `FindGMP.cmake`.

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-106.shtml

**Pull request merged**